src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -6,9 +6,27 @@ import { ContextConsumer, Mediator } from "Artsy/SystemContext"
 import { FollowArtistButtonFragmentContainer as FollowArtistButton } from "Components/FollowButton/FollowArtistButton"
 import React, { Component, Fragment } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
 import { Carousel } from "Styleguide/Components"
 import { Media } from "Utils/Responsive"
 
+/**
+ * This H1 and H2 were added for SEO purposes
+ */
+const H1 = styled.h1`
```

Comment:
Should this leave somewhere more global? We will probably have to do the same on Artwork page and other pages once we start rebuilding them in responsive.

Reply:
I think that's a great point.